packages/libxmlb.rb

```diff
@@ -0,0 +1,37 @@
+# Adapted from Arch Linux libxmlb PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/libxmlb/trunk/PKGBUILD
+
+require 'buildsystems/meson'
+
+class Libxmlb < Meson
+  description 'Library to help create and query binary XML blobs'
+  homepage 'https://github.com/hughsie/libxmlb'
+  version '0.3.14'
+  license 'LGPL'
+  compatibility 'x86_64 aarch64 armv7l'
```

Comment:
Does this fail to build on `i686`?

Reply:
The gtk deps imply gui, which doesn't work on i686. At this point not spending effort on customizations for i686 builds for GUI connected config options.

Reply:
Also, note that the gobject_instropection build dep for this isn't built for i686.

Reply:
Ended up building `gobject_introspection` for i686, after which I was then able to build `libxmlb` for i686!